node-installer.sh

```diff
@@ -170,20 +189,33 @@ update_pkg_database
 check_installed unzip unzip
 check_installed curl curl
 check_installed jq jq
-check_installed route net-tools
 check_installed logrotate logrotate
-check_installed dig dnsutils
-check_installed ufw ufw
 
-echo "Creating rusk service user"
-id -u dusk >/dev/null 2>&1 || useradd -r dusk
+# Ensure dusk group and user exist
+if ! id -u dusk >/dev/null 2>&1; then
+    echo "Creating dusk system user and group."
+    groupadd --system dusk
+    useradd --system --create-home --shell /usr/sbin/nologin --gid dusk dusk
+    echo "User 'dusk' and group 'dusk' created."
+else
+    echo "User 'dusk' and group 'dusk' already exist."
+fi
+
+# Add the current user to the dusk group for access
+echo "Adding current user to dusk group for access."
+if ! id -nG "$CURRENT_USER" | grep -qw "dusk"; then
+    usermod -aG dusk "$CURRENT_USER"
+    echo "User $CURRENT_USER has been added to the dusk group. Please log out and back in to apply changes."
```

Comment:
Does this means that, during an upgrade, people should logout before run "ruskreset"?